[MWAN/Transit] Added api key info GRE (#10268)
cloudflare/cloudflare-docs: content/magic-transit/_partials/_tunnel-endpoints.md

- **TTL**: Time to Live (TTL) in number of hops for the GRE tunnel. The default value is 64.
- **MTU**: Maximum Transmission Unit (MTU) in bytes for the GRE tunnel. The default value is 1476.

## Tunnels

### GRE tunnels

> You can set up GRE tunnels through the Cloudflare dashboard or via the API. However, if you want to use the API make sure you already have an [Account ID](/fundamentals/get-started/basic-tasks/find-account-and-zone-ids/) and [API Key](/fundamentals/api/get-started/keys/#view-your-global-api-key) before you begin.

Reviewer: ```suggestion
You can set up GRE tunnels through the Cloudflare dashboard or via the API. However, if you want to use the API, be sure to have your [account ID](/fundamentals/get-started/basic-tasks/find-account-and-zone-ids/) and [API key](/fundamentals/api/get-started/keys/#view-your-global-api-key) ready before you begin.
```
